pom.xml

```diff
@@ -467,8 +467,14 @@
 
       <dependency>
         <groupId>com.google.code.findbugs</groupId>
+        <artifactId>annotations</artifactId>
+        <version>${findbugs.version}</version>
+      </dependency>
+
+      <dependency>
+        <groupId>com.google.code.findbugs</groupId>
         <artifactId>jsr305</artifactId>
-        <version>${jsr305.version}</version>
+        <version>${findbugs.version}</version>
```

Comment:
We talked in person, so I understand the reason for the two deps, and their overlap, and why their order is necessary to satisfy `dependency:analyze`. Can you post `dependency:tree`, preferably a curated diff? This has been great for discussion of things like this in the past.